e2e-tests/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:2.7-alpine
+WORKDIR /src
+COPY requirements/requirements.txt /src
+RUN pip install pipenv
+RUN pipenv install --system -r requirements.txt --skip-lock --deploy
+COPY . /src
+CMD pytest --driver SauceLabs
```

Comment:
I would model this on the approach used by Mozillians, where the Docker image has Firefox and GeckoDriver installed, and can run the tests headlessly within the container.

Reply:
👍 to that suggestion

Reply:
@chartjes could that be done in a separate PR, or would you prefer/request that done here, in this one?

Reply:
On IRC, we agreed to leave it out for now, due to crashing Firefox - we'll test/file bugs/re-introduce when it's stable.